gradle/dependencies.gradle

```diff
@@ -31,6 +31,7 @@ ext {
       androidXAppCompatVersion  : '1.3.1',
       androidXPreferenceVersion : '1.1.0',
       androidXAnnotationVersion : '1.2.0',
+      androidXFragmentVersion   : '1.4.0',
```

Comment:
This is what caused a bunch of license changes. https://mvnrepository.com/artifact/androidx.fragment/fragment-ktx/1.4.0

For example, it's using an older version of lifecycle `2.3.1`.